src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2022 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <bitmap name="ScenesCopyMode" type="BITMAP8">
+    <cluster code="0x0005"/>
+    <field name="CopyAllScenes" mask="0x01"/>
+  </bitmap>
+
+  <struct name="AttributeValuePair">
+    <cluster code="0x0005"/>
+    <item name="attributeId" type="attrib_id" optional="true"/>
+    <item name="attributeValue" type="INT8U" array="true"/>
+  </struct>
+
+  <struct name="SceneExtensionFieldSet">
+    <cluster code="0x0005"/>
+    <item name="clusterId" type="cluster_id"/>
+    <item name="attributeValueList" type="AttributeValuePair" array="true"/>
```

Comment:
```suggestion
    <item name="ClusterId" type="cluster_id"/>
    <item name="AttributeValueList" type="AttributeValuePair" array="true"/>
```
to match the spec.

Reply:
I was under the impression we use lower Camel case in the code. But, yes in the spec document they are Upper Camel Case. I am changing all of them

Reply:
The code does different things in different places, but it always normalizes the case to what it wants (which can be UpperCamelCase or lowerCamelCase or a few other things).  

But for YAML we want to match the spec names, imo.